docs/tailwind.js

```diff
@@ -1,4 +1,5 @@
-var config = require('../defaultConfig')
+var config = require('../lib/index').defaultConfig()
+
```

Comment:
Was this extra line return really necessary Adam?

Reply:
Sorry dad, fixed.

Reply:
![thank you](https://user-images.githubusercontent.com/882133/32380725-2e08b84e-c087-11e7-84f0-702d0bd6aee1.gif)